config/vufind/searches.ini

```diff
@@ -261,8 +261,8 @@ WorkKeys = year
 ; EDSResultsDeferred:[GET parameter]:[result limit]
 ;       Same as EDSResults, but loaded via AJAX.
 ; EPFResults:[GET parameter]:[result limit]
-;       Display EPF (EBSCO Publication Finder) search results matching the terms 
-;       found in the specified GET parameter (default = "lookfor"), limited to a 
+;       Display EPF (EBSCO Publication Finder) search results matching the terms
+;       found in the specified GET parameter (default = "lookfor"), limited to a
```

Comment:
I assume sneaking in these whitespace fixes is intentional.  Sorry. :)

Reply:
Not a conscious decision, but my editor does it automatically and I decided it was harmless to include it here. :-)